_button.scss

```diff
@@ -0,0 +1,71 @@
+// button
+// ===
+
+// Configurable Variables
+// ---
+
+//Button Styles
+//---
+
+$button__padding: $vertical-input-padding $horizontal-input-padding !default;
+
+$button__border: if(variable-exists(border), $border, 1px solid $neutral-50) !default;
+$button__border-radius: if(variable-exists(border-radius), $border-radius, 0) !default;
+
+
+$button__font-color: inherit !default;
+$button__font-family: inherit !default;
+$button__font-size: inherit !default;
```

Comment:
I’m not sure we get a lot of value out of any of these variables. They all target a specific css property, and it’s not a lot of extra CSS to write project-level styles to override these. I think until we have a compelling need, we should keep configurable variables to a minimum.

That said, I think we may need to add one variable to stencil-variables to accomplish the goals for this component (see below)